plugins/nextjs/src/pageGenerator/index.ts

```diff
@@ -30,6 +30,13 @@ async function generatePage(inputOptions: Options) {
       name: 'itemName',
       message: 'Page name? (e.g. "SettingsPage")',
     },
+    {
+      type: 'input',
+      name: 'rootDir',
+      message:
+        'What is the root Directory for this? (default set to current working directory)',
+      default: process.cwd(),
+    },
```

Comment:
We should not need this. The code above should make sure that `rootDir` always exists.